packages/typography/Abbreviation.tsx

```diff
@@ -1,39 +1,36 @@
-import { h, Component, prop } from 'skatejs';
+import { h, Component } from 'skatejs';
 import styles from './Abbreviation.scss';
-import { css } from '@blaze-elements/common';
+import { shadyCssStyles } from '@blaze-elements/common';
 
-// public
-interface AbbreviationProps {
-  title: string,
-}
+export type AbbreviationProps = Props & EventProps;
 
-export class Abbreviation extends Component<AbbreviationProps> {
-  static get is() { return 'bl-abbreviation'; }
+export type AbbreviationAttrs = {
+  title?: string
+};
 
-  static get props() {
-    return {
-      title: prop.string( {
-        attribute: true
-      } )
-    };
-  }
+export type Props = {
+  title?: string
+};
+
+export type EventProps = {};
+
+export type AbbreviationEvents = {};
+
+@shadyCssStyles()
+export default class Abbreviation extends Component<AbbreviationProps> {
 
+  // FIX: enable projection from attribute
```

Comment:
This is same issue as in https://github.com/wc-catalogue/blaze-elements/pull/275#discussion_r105011772